CHANGELOG.md

```diff
@@ -4,13 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-- No changes yet.
+## [1.8.0] - 2019-03-27
+### Added
+- Added `As` option for `Provide` to specify that the value produced by the
```

Comment:
Confusing sentence. Not sure if I can make it more clear, though. 

Perhaps something like?

> Added `As` option for `Provide` to allow constructor functions explicit control over which types they are being inserted into the graph for